### ISSUE TITLE
VS Code Extension Broken

### ISSUE THREAD

**Issue by jr-dev1001** [Exafunction/codeium#93]
I'm facing the issue since yesterday(18/09/2024) with VS Code extension , nothing is supporting even the codeium chat, autocomplete, generation and explain. VS code extension broken, I also tried to switching to pre-released and released versions but issue remains the same. The (ALT+C ALT+R) aren't much useful and chat is forever loading (unresponsive). 

![image](https://github.com/user-attachments/assets/9f9d4136-8e6d-4e82-95dd-6394dfb1c878)

**Comment by zocky** [comment]
I've also had an issue using codeium chat in VSCode since yesterday.

This seems to be the relevant section from the diagnostics:
```
      "W0920 23:47:40.942673 1425581 chat_token_limit.go:53] error converting token count to int: experiment key CHAT_TOKENS_SOFT_LIMIT does not have variant payload\n",
      "W0920 23:47:42.283428 1425581 chat_token_limit.go:53] error converting token count to int: experiment key CHAT_TOKENS_SOFT_LIMIT does not have variant payload\n",
      "E0920 23:47:42.293465 1425581 interceptor.go:51] /exa.language_server_pb.LanguageServerService/GetChatMessage: insufficient space for system prompt and completion tokens. Soft limit: 2900, hard limit: 13457, generation reservation: 2048, system prompt tokens: 879\n",
      "W0920 23:47:42.294239 1425581 api.go:160] &{0xc00101a408}.GetChatMessage(_) = _, rpc error: code = Unknown desc = insufficient space for system prompt and completion tokens. Soft limit: 2900, hard limit: 13457, generation reservation: 2048, system prompt tokens: 879\n",
      "W0920 23:47:42.294312 1425581 api.go:160] &{0xc00101a408}.GetChatMessage(_) = _, rpc error: code = Unknown desc = insufficient space for system prompt and completion tokens. Soft limit: 2900, hard limit: 13457, generation reservation: 2048, system prompt tokens: 879\n",
      "W0920 23:47:42.294369 1425581 api.go:160] &{0xc00101a408}.GetChatMessage(_) = _, rpc error: code = Unknown desc = insufficient space for system prompt and completion tokens. Soft limit: 2900, hard limit: 13457, generation reservation: 2048, system prompt tokens: 879\n",
      "W0920 23:47:42.294422 1425581 api.go:160] &{0xc00101a408}.GetChatMessage(_) = _, rpc error: code = Unknown desc = insufficient space for system prompt and completion tokens. Soft limit: 2900, hard limit: 13457, generation reservation: 2048, system prompt tokens: 879\n",
      "W0920 23:47:42.294475 1425581 api.go:160] &{0xc00101a408}.GetChatMessage(_) = _, rpc error: code = Unknown desc = insufficient space for system prompt and completion tokens. Soft limit: 2900, hard limit: 13457, generation reservation: 2048, system prompt tokens: 879\n",
      "W0920 23:47:42.294480 1425581 api.go:163] Clearing chat history due to error: rpc error: code = Unknown desc = insufficient space for system prompt and completion tokens. Soft limit: 2900, hard limit: 13457, generation reservation: 2048, system prompt tokens: 879\n",
```

**Comment by jr-dev1001** [comment]
For quick fix down grade the version to 1.8.x , I don't know why but even 1.10.x is also not working. I raised the same issue in  [Exafunction/CodeiumVisualStudio](https://github.com/Exafunction/CodeiumVisualStudio/issues/121) and closing it here.
